app/models/file.js

```diff
@@ -1,11 +1,16 @@
 import DS from 'ember-data';
-const { Model } = DS;
+const { Model, attr } = DS;
 
-export default Model.extend({
-  name: DS.attr('string'),
-  externalId: DS.attr('string'),
-  description: DS.attr('string'),
-
-  /** processing, changesRequired, accepted, published */
-  status: DS.attr('string')
-});
+export default class File extends Model {
+  @attr('string') name;
```

Comment:
Nice :) I had not even thought to upgrade the Model files to class and decorator syntax